src/main/java/com/marizueva/laboratory/hw6/forms/MetalAndColorForm.java

```diff
@@ -0,0 +1,64 @@
+package com.marizueva.laboratory.hw6.forms;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.jdi.tools.DataClass;
+import com.marizueva.laboratory.hw6.entities.MetalAndColorData;
+
+public class MetalAndColorForm extends DataClass<MetalAndColorForm> {
```

Comment:
This class should extend Form with DataClass
For an example check LoginForm

Reply:
fixed